### DISCUSSION TITLE
saitek heavy machinary wheels/pedal/side panel

### DISCUSSION THREAD

**Discussion by becherbrook** [berarma/oversteer#216] [Q&A]
Hi, I've been having issues generally getting this to work correctly in Fs22 because it seems to be locked on single axis (only in game, jstestgsk recognises everything fine), and was hoping oversteer might be able to fix the issue for me. I realise I might be barking up the wrong tree entirely, but interestingly enough Oversteer doesn't even recognise/see my hardware at all even though Linux/jstestgsk DOES.

Is there a solution here?

**Comment by berarma** [comment]
I'm sorry, but Oversteer doesn't fix issues in games. Recognizing the wheel will only be useful to test it the same way you're already testing it with jstest.

**Reply by becherbrook** [reply]
So I actually solved this if its of any interest, there is apparently a long standing issue with steam's soldier and gaming wheels that seems to effect proton 7 and beyond. I reverted back to 6.3 and it works.
